### PR TITLE
Issue/cleanupcode/76

### LINKED ISSUE #76: Cleanup Code

### PR DESCRIPTION
**Pull request by lmtjanssen**
Fixed #76

### CODE REVIEW

File: src/views/user/ProfileUserView.jsx [hunk 1]
```diff
@@ -63,23 +62,23 @@ class MyProfile extends Component {
...
+                this.userNames = userNames;
+                console.log(this.userNames)
             })
     }
 
-    checkAvailibility(username) {
-        console.log(this.Usernames)
-        return !this.Usernames.find(u => u === username);
+    checkAvailability(username) {
+        console.log(this.userNames);
```
Comment (larsvandriel) [inline src/views/user/ProfileUserView.jsx]: Is it really neccessary to log the username? Isn't this just for testing?

File: src/views/user/ProfileUserView.jsx [hunk 2]
```diff
@@ -88,13 +87,13 @@ class MyProfile extends Component {
     }
 
     saveUsername() {
-        const newusername = this.state.NewUsername;
-        console.log(this.checkAvailibility(newusername))
-        if (this.checkAvailibility(newusername)) {
+        const newUserName = this.state.NewUsername;
+        console.log(this.checkAvailability(newUserName));
```
Comment (larsvandriel) [inline src/views/user/ProfileUserView.jsx]: Another console log that we might delete.

File: src/views/user/ProfileUserView.jsx [hunk 3]
```diff
@@ -88,13 +87,13 @@ class MyProfile extends Component {
     }
 
     saveUsername() {
-        const newusername = this.state.NewUsername;
-        console.log(this.checkAvailibility(newusername))
-        if (this.checkAvailibility(newusername)) {
+        const newUserName = this.state.NewUsername;
```
Comment (larsvandriel) [inline src/views/user/ProfileUserView.jsx]: Shouldn't this.state.NewUsername also be camelCase?

File: src/views/user/ProfileUserView.jsx [hunk 4]
```diff
@@ -117,4 +116,4 @@ class MyProfile extends Component {
     }
 }
 
-export default MyProfile
+export default ProfileUserView;
```
Comment (larsvandriel) [inline src/views/user/ProfileUserView.jsx]: Can't you just export the class at the top. So

export default class ProfileUserView extends Component

File: src/views/team/CreateTeamView.jsx [hunk 5]
```diff
@@ -55,4 +55,4 @@ class CreateTeam extends Component {
   }
 }
 
-export default CreateTeam;
```
[the remark this answers is not in the document]
Comment (lmtjanssen) [reply src/views/team/CreateTeamView.jsx]: This is the default syntax for React as far as I could find

File: src/views/firebase/FirebaseAuthView.jsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+        callbacks: {
+            signInSuccessWithAuthResult : () => false
+        }
+    };
+
+    componentDidMount() {
+        firebase.auth().onAuthStateChanged(user => {
+            if (user != null) {
+                ApiCommunication.graphQlCallPost("mutation m($uuid: String!) { createUser(uuid: $uuid) { id } }",`{"uuid": "${user.uid}"}`)
+                  .then(res => console.log(res))
```
[the remark this answers is not in the document]
Comment (lmtjanssen) [reply src/views/firebase/FirebaseAuthView.jsx]: Got rid of console.log

File: src/views/user/ProfileUserView.jsx [hunk 7]
```diff
@@ -63,23 +62,23 @@ class MyProfile extends Component {
...
                 snapshot.forEach(doc => {
                     const data = doc.data();
-                    usernames.push(data.username)
+                    userNames.push(data.username)
                 });
             }).then(() => {
-                this.Usernames = usernames  
-                console.log(this.Usernames)
+                this.userNames = userNames;
+                console.log(this.userNames)
```
[the remark this answers is not in the document]
Comment (lmtjanssen) [reply src/views/user/ProfileUserView.jsx]: It is not, removed both

File: src/views/user/ProfileUserView.jsx (hunk 2 above)
Comment (lmtjanssen) [reply src/views/user/ProfileUserView.jsx]: Removed it

File: src/views/user/ProfileUserView.jsx (hunk 3 above)
Comment (lmtjanssen) [reply src/views/user/ProfileUserView.jsx]: It should

**Pull request merged**